openlibrary/plugins/openlibrary/js/ol.js

```diff
@@ -354,23 +354,22 @@ export function initReadingListFeature() {
     /* eslint-disable no-unused-vars */
     // success function receives data on successful request
     $(document).on('change', '.reading-log-lite select', function(e) {
-        var self = this;
-        var form = $(self).closest('form');
-        var remove = $(self).children('option').filter(':selected').text().toLowerCase() === 'remove';
-        var url = $(form).attr('action');
+        const self = this,
+            form = $(self).closest('form');
```

Comment:
This is only used once; let's just inline it. It doesn't contribute to readability any.

```suggestion
```

Reply:
Done